user/templates/profile/profile-beta.html

```
@@ -44,8 +44,11 @@
                             <img id="profile-photo" class="w-48 h-48 mb-3 rounded-full object-cover shadow-lg"
                                 src="{{ user.profile.photo.url }}" alt="{{user.first_name}} profile">
                         {% else %}
-                            <img id="profile-photo" class="w-48 h-48 mb-3 rounded-full shadow-lg"
-                                src="https://flowbite.com/docs/images/people/profile-picture-3.jpg" alt="default user photo">
+                            <div id="profile-photo" class="w-48 h-48 mb-3 rounded-full shadow-lg ">
```

HTMLHint (1.5.0)

[error] The id value [ profile-photo ] must be unique.

(id-unique)